.harness/solutions/2026-05-13-jsc-321-context-pack-smoke-path-contract.md

```
@@ -0,0 +1,212 @@
+---
+schema_version: 1
+artifact_id: jsc-321-context-pack-smoke-path-contract-solution
+artifact_type: he-compound-reinforce-solution
+canonical_slug: jsc-321-context-pack-smoke-path-contract
+title: JSC-321 Context-Pack Smoke Path Contract Solution
+harness_stage: he-compound
+status: active
+date: 2026-05-13
+refreshed_date: 2026-05-14
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)